### DISCUSSION TITLE
如何使用 http 接口获取 milvus 的运行状态？

### DISCUSSION THREAD

**Discussion by ponponon** [milvus-io/milvus#26737] [Q&A and General discussion]
如何使用 http 接口获取 milvus 的运行状态？比如有几个集合，每个集合的状态（是否导入，导入中的话，导入了多少百分比）、集合中的字段等等，就像 attu 一样，有相关接口文档吗？

**Comment by ponponon** [comment]
attu 获取 milvus 的信息，用的是 19530 端口还是 9091 ？

**Reply by yhmo** [reply]
主要的sdk文档都在官网里：https://milvus.io/docs     API Reference之下
Attu使用的是Node SDK，以RPC的方式访问milvus的19530端口

**Comment by xiaofan-luan** [comment]
我们正在改进Attu的能力，需要获取什么信息可以一起讨论
